lib/settings_base.py

```diff
@@ -80,6 +80,9 @@ def path(*folders):
 DATABASES['default']['TEST_CHARSET'] = 'utf8'
 DATABASES['default']['TEST_COLLATION'] = 'utf8_general_ci'
 
+# Run all views in a transaction unless they are decorated not to.
+ATOMIC_REQUESTS = True
```

Comment:
Database is supposed to be pinned to master after a write, through our db routing app. Since the setting is per database we only need it on the master.